### PR TITLE
Update cargo flags

### LINKED ISSUE #29: Cargo flags on installation page are wrong
On the installation page it says to run `cargo install nu --all --features=stable` for an "all bells and whistles" install. But it seems there is no `--all` flag in cargo. I'm new to Rust so not sure if this should be `--all-features` or `--features=stable` which both seem valid.
EDIT: I have copied the installation instructions from the book

### PR DESCRIPTION
**Pull request by nespera**
I changed them to match what is in the book. Should fix #29

### CODE REVIEW

**Comment by waldyrious** [comment]
Cross-referencing [related discussion](https://github.com/nushell/nushell.github.io/pull/28#discussion_r368233299). Note that any changes to the flags should be reflected in the [main repo's README](https://github.com/nushell/nushell#local) as well.

**Comment by jonathandturner** [comment]
I believe the line we should be using going forward is:

```
cargo build --workspace --features=stable
```

**Comment by nespera** [comment]
I don't know much about cargo, but it looks like in the previous change the `--all` flag was added to `cargo install` which does not support it, unlike `cargo build`, which does.

File: installation.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ From there, we can build and run Nu if we are using `rustup` with:
 
 ```
 > cd nushell
-nushell> cargo build --all --features=stable && cargo run --all --features=stable
+nushell> cargo build --all-features && cargo run --all-features
```
Comment (andrasio) [inline installation.md]: This won't work correctly. Basically `--all-features` also adds the tracer which we need to fix. For `cargo build` and `cargo run` we better leave them as `--all --features=stable`

File: installation.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ You can also install Nu with all the bells and whistles (be sure to have install
 [dependencies](https://book.nushell.sh/en/installation#dependencies) for your platform):
 
 ```
-cargo install nu --all --features=stable
+cargo install nu --features=stable
```
Comment (andrasio) [inline installation.md]: Good.

File: installation.md (hunk 1 above)
Comment (nespera) [reply installation.md]: --all is not supported by `cargo run` as far as I know

Comment (andrasio) [reply installation.md]: omg, you are right. Then we may want to have `cargo run --features=stable` (not `--all-features`)

Comment (andrasio) [reply installation.md]: For the case of `cargo build` I also agree with @jonathandturner 's comment.

Comment (nespera) [reply installation.md]: Ok, updated to use @jonathandturner 's suggestion for `cargo build` and yours for `cargo run`

**Comment by sebastian-xyz** [comment]
Thank you!

**Pull request merged**